src/common/cockpitunixsignal.c

```diff
@@ -67,7 +67,7 @@ struct signv {
   { "PIPE",     SIGPIPE },      /* 13 */
   { "ALRM",     SIGALRM },      /* 14 */
   { "TERM",     SIGTERM },      /* 15 */
-#ifndef _MIPS_ARCH
+#if !defined(__alpha__) && !defined(__mips__) && !defined(__sparc__)
```

Comment:
Actually, I wonder if this could simply use `#ifdef SIGSTKFLT` instead of hardcoding the architectures?